README.md

```diff
@@ -105,7 +105,7 @@ Monitors jobs launched by the [Cromwell workflow engine](https://github.com/broa
 #### Notes
 1. Websocket reload on code change does not work in docker-compose (see
 https://github.com/angular/angular-cli/issues/6349).
-2. Changes to `package.json` or `requirements.txt` or [regenerating the API](#updating-the-api-using-swagger-codegen) require a rebuild with:
+2. Changes to `package.json`, `requirements.txt` or `constraints.txt` [regenerating the API](#updating-the-api-using-swagger-codegen) require a rebuild with:
```

Comment:
Happy to go with your preference here but can't miss this chance to stand up for the oxford comma. 

![](https://img1.wsimg.com/isteam/ip/ad62444f-c469-4bef-8120-7569c68bf266/Oxford%20Comma.jpg)